domain-xample/svc-xample-j/src/integrationTest/java/gov/va/vro/services/xample/XampleJavaMicroserviceTest.java

```diff
@@ -0,0 +1,91 @@
+package gov.va.vro.services.xample;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import gov.va.vro.model.xample.SomeDtoModel;
+import gov.va.vro.model.xample.StatusValue;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.amqp.core.Message;
+import org.springframework.amqp.core.MessageProperties;
+import org.springframework.amqp.rabbit.core.RabbitAdmin;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.io.IOException;
+
+import static gov.va.vro.services.xample.JavaMicroserviceApplication.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+/**
+ * Start RabbitMQ:
+ * docker compose up -d rabbitmq-service
+ * <p>
+ * Run these tests:
+ * ./gradlew :domain-xample:svc-xample-j:integrationTest
+ */
+@SpringBootTest
+@Slf4j
+public class XampleJavaMicroserviceTest {
+
+  @Autowired
+  private RabbitTemplate rabbitTemplate;
```

Comment:
Don't need to create separate MQ producer and consumer if using Spring's `RabbitTemplate`.

Reply:
🤯